Misc/NEWS.d/next/Core and Builtins/2023-05-17-19-54-22.gh-issue-71679.-nD6eA.rst

```diff
@@ -0,0 +1 @@
+Enhance :meth:`bytearray.__repr__` with :meth:`bytes._repr__`'s logic. Patched by Xiang Zhang and Furkan Onder.
```

Comment:
I think this can be clearer.

```suggestion
Improve the performance of :meth:`bytearray.__repr__` by utilizing :meth:`bytes._repr__`. Patched by Xiang Zhang and Furkan Onder.
```